Userland/Applications/PixelPaint/Image.cpp

```diff
@@ -212,6 +213,21 @@ ErrorOr<void> Image::export_png_to_file(Core::File& file, bool preserve_alpha_ch
     return {};
 }
 
+ErrorOr<void> Image::export_qoi_to_file(Core::File& file, bool preserve_alpha_channel) const
+{
+    // TODO: Handle QOI RGB encoding
+    if (!preserve_alpha_channel)
```

Comment:
I get that this is an existing thing, but why would anyone ever want to do that... and what happens to values with opacity? :shrug:

Reply:
I think it is useful to have an explicit option to not preserve opacity.

Reply:
You're right. I changed it to always export as RGBA